src/ui/components/address-action/AddressActionDetails/AddressActionDetails.tsx

```diff
@@ -36,10 +38,11 @@ export function AddressActionDetails({
         <RecipientLine
           recipientAddress={recipientAddress}
           chain={chain}
+          showNetworkIcon={!showApplicationLine}
```

Comment:
We want to display network icon in recipient line only if we are not displaying application line

Reply:
Should it be `!(showApplicationLine && addressAction?.label)` then?

Reply:
Seems so, though I would extract a helper variable

Reply:
Yeah, that would be more correct, thank you ✅